consensus/consensus.go

```diff
@@ -200,7 +196,7 @@ func (consensus *Consensus) GetPrivateKeys() multibls.PrivateKeys {
 }
 
 // GetLeaderPrivateKey returns leader private key if node is the leader
-func (consensus *Consensus) GetLeaderPrivateKey(leaderKey *bls_core.PublicKey) (*bls.PrivateKeyWrapper, error) {
+func (consensus *Consensus) getLeaderPrivateKey(leaderKey *bls_core.PublicKey) (*bls.PrivateKeyWrapper, error) {
```

Comment:
Maybe change the name of this function to `getPrivateKeyFromPublic`

Reply:
I don't see any advantages or disadvantages from this renaming, i will leave it as is.